infra/charts/management-service/values.yaml

```diff
@@ -55,7 +55,9 @@ xpManagement:
     class: ""
 
   # -- XP Management Service server configuration.
-  apiConfig: {}
+  apiConfig:
```

Comment:
Rather than doing this, the helper can check if `sentryConfig` is also set first. See: https://stackoverflow.com/a/68807258